src/Components/StopWatch/StopWatch.tsx

```diff
@@ -4,8 +4,8 @@ import ControlButtons from "../ControlButton";
 import Timer from "../Timer";
   
 function StopWatch() {
-  const [isActived, setIsActived] = useState(false);
-  const [isPaused, setIsPaused] = useState(true);
+  const [isActived, setIsActived] = useState(false); // For Check Press Reset Button
+  const [isPaused, setIsPaused] = useState(true); // For Check Pause
```

Comment:
Same here. I have no idea why we need `isPaused`,
We can just have `isActive` (`isActived` is a wrong name) state. And if a user click 'Reset', you just need to set isActive as false and timer as zero.